templates/telemetry_derived/bq_event_events/init.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE IF NOT EXISTS
+  `moz-fx-data-shared-prod.telemetry_derived.bq_event_events` (
```

Comment:
`event_events` oh how i wish there were a better name for this, but alas I can't think of one v_v

Reply:
Well, the good news is that users will never see the table name